ajax/libs/redux-search/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "redux-search",
+  "filename": "umd/redux-search.js",
```

Comment:
Let's use `umd/redux-search.min.js`
We provide minified file as the file name, the current file is not minified.
Thanks.

Reply:
Done, @sufuf3 please confirm the changes, thank you!